src/library/browse/browsefeature.cpp

```diff
@@ -284,6 +285,36 @@ void BrowseFeature::onRightClickChild(const QPoint& globalPos, QModelIndex index
      onLazyChildExpandation(index);
 }
 
+namespace {
```

Comment:
Please rename the function to just `getRemovableDevices()` and enclose the whole code in `#if defined(__LINUX__)`.

Reply:
Done.